### PR TITLE
(Block) entity position fixes (#964, #998, #1046, #1061)

### PR DESCRIPTION
**Pull request by YanisBft**

### CODE REVIEW

File: mappings/net/minecraft/entity/Entity.mapping [hunk 1]
```diff
@@ -202,7 +202,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	METHOD method_23325 getParticleZ (D)D
 		ARG 1 widthScale
 	METHOD method_23326 getVelocityMultiplier ()F
-	METHOD method_23327 setPos (DDD)V
+	METHOD method_23327 setPosition (DDD)V
```
Comment (liach) [inline mappings/net/minecraft/entity/Entity.mapping]: pos imo is fine (shorter)

File: mappings/net/minecraft/block/entity/BlockEntity.mapping [hunk 2]
```diff
@@ -25,7 +25,7 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
-	METHOD method_11009 setWorld (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)V
+	METHOD method_11009 setLocation (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;)V
```
Comment (liach) [inline mappings/net/minecraft/block/entity/BlockEntity.mapping:28]: is this more like init location (i.e. only called once in lifetime or after revalidation)

File: mappings/net/minecraft/entity/Entity.mapping (hunk 1 above)
Comment (YanisBft) [reply mappings/net/minecraft/entity/Entity.mapping]: Do I rename `Position` to `Pos` in all methods?

File: mappings/net/minecraft/block/entity/BlockEntity.mapping (hunk 2 above)
Comment (YanisBft) [reply mappings/net/minecraft/block/entity/BlockEntity.mapping:28]: It's called each time `setBlockEntity` is called, so I think `set` is better than `init` in this case

Comment (Juuxel) [reply mappings/net/minecraft/block/entity/BlockEntity.mapping:28]: Isn't `setBlockEntity` only called once per BE?

Comment (YanisBft) [reply mappings/net/minecraft/block/entity/BlockEntity.mapping:28]: Nope it's not. Additionally, `setLocation` is called in `loadBlockEntity`

File: mappings/net/minecraft/entity/Entity.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/entity/Entity.mapping]: Hmm, good question. This method compared to others is fairly frequently used as all xyz direct modifications has been migrated to this method in 1.15 snapshots.

**Review by liach — APPROVED** [review]

**Comment by liach** [comment]
Lgtm. hope we can get this fix in soon.

**Review by Juuxel — APPROVED** [review]

**Pull request merged**